website/source/docs/agent/options.html.markdown

```diff
@@ -115,8 +115,9 @@ will exit with an error at startup.
   [`-bind` command-line flag](#_bind), and if this is not specified, the `-bind` option is used. This is available in Consul 0.7.1 and later.
 
 * <a name="_client"></a><a href="#_client">`-client`</a> - The address to which
-  Consul will bind client interfaces, including the HTTP and DNS servers. By default,
-  this is "127.0.0.1", allowing only loopback connections.
+  Consul will bind client interfaces, including the HTTP and DNS servers. When
```

Comment:
Can you qualify these with "in Consul 0.8 or later this is X, and previously it was Y".